connect/cli/plugins/project/renderer.py

```diff
@@ -0,0 +1,50 @@
+import os
+from string import Template
+
+from jinja2 import Environment, FileSystemLoader, select_autoescape
+
+from connect.cli.core.terminal import console
+
+
+class BoilerplateRenderer:
```

Comment:
Probably we should create directories here using also paths with variables and using string template to replace placeholders with values.